.storybook/preview-head.html

```diff
@@ -0,0 +1,5 @@
+<style>
+  body {
+    overflow: scroll;
```

Comment:
This change is needed due to [this](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/presentation/main.less#L1162-L1164) which prevents scrolling the storybook content